test/e2e/memcached_test.go

```diff
@@ -118,7 +142,7 @@ func TestMemcached(t *testing.T) {
 				t.Fatal(err)
 			}
 			// TODO: make this match more complete in case we add another repo tracking master
-			gopkg = bytes.Replace(gopkg, []byte("branch = \"master\""), []byte("# branch = \"master\""), -1)
+			gopkg = bytes.Replace(gopkg, []byte("branch = \"refactor/controller-runtime\""), []byte("# branch = \"refactor/controller-runtime\""), -1)
```

Comment:
This should be matched to the branch that we are currently on. These branches are close enough that they are not breaking, but we should probably switch this back to master once we merge this PR and then figure out how to handle this best (this will become a bit more of a problem when we start having release branches).

Reply:
This actually is a change that has to happen in 2 places: here and in the gopkg toml scaffold. Not changing this shouldn't break the PR tests since we're just commenting out the branch, but it would break the master branch and users if not changed.

Reply:
After reverting this and merging, I can write a quick PR to update any branch with the branch annotation I added.